user-routes.js

```diff
@@ -35,13 +49,28 @@ app.post('/users', function(req, res) {
 });
 
 app.post('/sessions/create', function(req, res) {
-  if (!req.body.username || !req.body.password) {
+
+  var username;
+  var userScheme;
+  
+  // The POST contains a username and not an email
+  if(req.body.username) {
```

Comment:
This code is repeated. Can we extract it to some function or something similar? Otherwise, let's create 2 different endpoints for Email and Username signup and login. What do you think?